pkg/controllers/raycluster_webhook.go

```diff
@@ -77,6 +81,30 @@ func (w *rayClusterWebhook) Default(ctx context.Context, obj runtime.Object) err
 		rayCluster.Spec.HeadGroupSpec.Template.Spec.ServiceAccountName = rayCluster.Name + "-oauth-proxy"
 	}
 
+	// add default queue label if not present
+	if rayCluster.GetLabels() == nil {
+		rayCluster.Labels = make(map[string]string)
+	}
+	err := w.Client.List(ctx, &kueuev1beta1.LocalQueueList{})
+	if err != nil {
+		rayclusterlog.Error(err, "Failed to list LocalQueues, Kueue CRD might not be installed")
+		_, ok := rayCluster.Labels["kueue.x-k8s.io/queue-name"]
+		if !ok {
+			// check if CRD Kueue LocalQueue exists
+			localQueues := &kueuev1beta1.LocalQueueList{}
+			err := w.Client.List(ctx, localQueues)
+			if err == nil {
+				for _, localQueue := range localQueues.Items {
+					is_default, ok := localQueue.Labels["kueue.x-k8s.io/default-queue"]
```

Comment:
We should use something like `kueue.openshift.ai/default-queue` instead. We can also explore the idea upstream in parallel.

Reply:
This label is already being used in CF-sdk I'm not sure when it was added, but changing it could break users